Neuron/Sections/Settings/View/AboutUsTableViewController.swift

```diff
@@ -17,20 +18,17 @@ class AboutUsTableViewController: UITableViewController {
         setVersionLabel()
     }
 
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        let urlString: String = URLs[segue.identifier ?? ""] ?? URLs.first!.value
-        let webViewController = segue.destination as! CommonWebViewController
-        webViewController.url = URL(string: urlString)!
-    }
+    private var firstSectionUrls = [
```

Comment:
The mapping idea is great! But the names could be more meaningful. For example, instead of `secondSectionUrls`, use `serviceUrls`.